linkage-checker-exclusion-rules.xml

```diff
@@ -7,6 +7,14 @@
     <Target><Package name="org.conscrypt"/></Target>
     <Reason>Optional</Reason>
   </LinkageError>
+  <LinkageError>
+    <Target><Package name="com.google.crypto.tink.subtle.X25519"/></Target>
+    <Reason>Optional</Reason>
+  </LinkageError>
+  <LinkageError>
+    <Target><Package name="com.google.crypto.tink.subtle"/></Target>
+    <Reason>Optional</Reason>
+  </LinkageError>
```

Comment:
With L14-17, L10-13 can be removed as its redundant. Packge includes everything under.